R/transformCounts.R

```diff
@@ -307,6 +346,120 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
     return(mat)
 }
 
+####################################.calc_css###################################
+# This function applies cumulative sum scaling (CSS) to the abundance table.
+.calc_css <- function(mat, css_percentile = NULL, ...) {
+    # Check for the presence of the scaling factor percentile and scaling 
+    # constant in additional args
+    if (is.null(css_percentile)) {
+      css_percentile <- ifelse("css_percentile" %in% names(list(...)), 
+                               list(...)$css_percentile, 0.5)
+    }
```

Comment:
What is the thing with this? css_percentile should already catch the value in the function definition?